tests/MongoDB.Driver.Core.TestHelpers/CoreTestConfiguration.cs

```diff
@@ -124,8 +124,8 @@ public static ClusterBuilder ConfigureCluster(ClusterBuilder builder)
                         {
                             cert = new X509Certificate2(certificateFilename, password);
                         }
-                        return ssl.With(
```

Comment:
To have a more accurate git history, it looks like the changes in this file can be reverted, since no real logic has been changed

Reply:
Done

Reply:
Nope :) I meant that all changes in this file can be reverted since now there are only formatting changes in this file

Reply:
Oops the revert didn't work hm trying again!